src/quequations.jl

```diff
@@ -15,18 +15,22 @@ end
 @doc """
 Schrodinger Equation method
 
-Input Parameters :
+### Arguments
 
-`hamiltonian` : Hamiltonian of the system to construct `QuSchrodingerEq` type.
+Inputs :
+* hamiltonian :: Hamiltonian of the system
+
+Output :
+* QuSchrodingerEq type construct.
 """ ->
 QuSchrodingerEq{H<:QuBase.AbstractQuMatrix}(hamiltonian::H) = QuSchrodingerEq{H}(hamiltonian)
 
 @doc """
 Liouville von Neumann Equation type
 
-Fields :
+### Fields :
 
-`liouvillian` : Liouvillian of the system is the only field for the type.
+liouvillian : Liouvillian of the system is the only field for the type.
```

Comment:
You can omit "is the only field for the type." as you don't say this for `QuSchrodingerEq`.